tests/test_pixel_inflation.py

```diff
@@ -0,0 +1,306 @@
+    if sequence_fixture == "not_a_sequence":
+        sequence = []
+    else:
+        sequence = request.getfixturevalue(sequence_fixture) if sequence_fixture is not None else None
+
+    if expect_error:
+        # Expect a specific exception (either ValueError or RuntimeError)
+        with pytest.raises(expected_exception):
+            pi.find_pixel_inflations(mock_eit_data, mock_continuous_data, sequence, store=store_input)
+    else:
+        # Run pixel inflation detection and check the result
+        result = pi.find_pixel_inflations(mock_eit_data, mock_continuous_data, sequence, store=store_input)
+
+        # If store is True or None and sequence is not None, check that the result is stored in the sequence
+        if (store_input is True or store_input is None) and sequence is not None:
+            assert len(sequence.interval_data.data) == 1
+            assert sequence.interval_data.data[0] == result
+        elif sequence is not None:
+            assert len(sequence.interval_data.data) == 0
+
+
+def mock_compute_pixel_parameter(mean: int):
+    def _mock(*_args, **_kwargs) -> np.ndarray:
+        if mean < 0:
+            return np.array([[[-1, -1], [-1, -1]], [[-1, -1], [-1, -1]]])
+        if mean > 0:
+            return np.array([[[1, 1], [1, 1]], [[1, 1], [1, 1]]])
+        return np.array([[[0.0, 0.0], [0.0, 0.0]], [[0.0, 0.0], [0.0, 0.0]]])
+
+    return _mock
+
+
+@pytest.mark.parametrize(
+    ("mean"),
+    [
+        0,
+        -1,
+        1,
+    ],
+)
+def test_with_custom_mean_pixel_tiv(mock_eit_data: MockEITData, mock_continuous_data: MockContinuousData, mean: int):
+    mock_function = mock_compute_pixel_parameter(mean)
+    with patch(
+        "eitprocessing.parameters.tidal_impedance_variation.TIV.compute_pixel_parameter",
+        side_effect=mock_function,
+    ):
+        pi = PixelInflation(breath_detection_kwargs={"minimum_duration": 0.01})
+
+        result = pi.find_pixel_inflations(mock_eit_data, mock_continuous_data)
+
+        assert result.values.shape == (3, 2, 2)
+
+        if mean == 0:
+            pass
+        else:
+            for row in range(2):
+                for col in range(2):
+                    if mean == 0:
+                        # Expect None values when mean == 0
+                        assert result.values[1, row, col] is None
+                    else:
+                        time_point = result.values[1, row, col].middle_time
+                        index = np.where(mock_eit_data.time == time_point)[0]
+                        value_at_time = mock_eit_data.pixel_impedance[index[0], row, col]
+                        if mean == -1:
+                            assert math.isclose(value_at_time, -1, abs_tol=0.01)
+                        elif mean == 1:
+                            assert math.isclose(value_at_time, 1, abs_tol=0.01)
+
+
+def test_with_data(draeger1: Sequence, draeger2: Sequence, timpel1: Sequence, pytestconfig: pytest.Config):
```

Comment:
This last test is **very** slow and will slow down continuous integration on the server. All tests are run on every push on a pull request, on every merge, et cetera. 

I think it's important to test whether the overal structure works on real data, but maybe it's better to only one dataset, or just a small part (e.g. 20 breaths) of a larger dataset.

Reply:
Tests are sufficiently fast now, I think.